### DISCUSSION TITLE
Hello I would like to ask for help, Im facing Error: [11620] Failed to execute script chia

### DISCUSSION THREAD

**Discussion by userbox020** [swar/Swar-Chia-Plot-Manager#589] [Q&A]
[11620] Failed to execute script chia
Traceback (most recent call last):
  File "chia\cmds\chia.py", line 81, in <module>
  File "chia\cmds\chia.py", line 77, in main
  File "click\core.py", line 829, in __call__
  File "click\core.py", line 782, in main
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1259, in invoke
  File "click\core.py", line 1066, in invoke
  File "click\core.py", line 610, in invoke
  File "click\decorators.py", line 21, in new_func
  File "chia\cmds\plots.py", line 135, in create_cmd
  File "chia\plotting\create_plots.py", line 55, in create_plots
ValueError: Length of bytes object not equal to G1Element::SIZE

**Comment by userbox020** [comment]
--------------------------------- This is my config.yaml

chia_location: C:\Users\ADMIN01\AppData\Local\chia-blockchain\app-1.1.5\resources\app.asar.unpacked\daemon\chia.exe

manager:
  check_interval: 60
  log_level: ERROR


log:
  folder_path: e:\Chia\Logs\Plotter


view:
  check_interval: 60
  datetime_format: "%Y-%m-%d %H:%M:%S"
  include_seconds_for_phase: false
  include_drive_info: true
  include_cpu: true
  include_ram: true
  include_plot_stats: true


notifications:

  notify_discord: false
  discord_webhook_url: https://discord.com/api/webhooks/0000000000000000/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

  notify_sound: false
  song: audio.mp3

  notify_pushover: false
  pushover_user_key: xx
  pushover_api_key: xx
  
  notify_twilio: false
  twilio_account_sid: xxxxx
  twilio_auth_token: xxxxx
  twilio_from_phone: +1234657890
  twilio_to_phone: +1234657890


progress:
  phase1_line_end: 801
  phase2_line_end: 834
  phase3_line_end: 2474
  phase4_line_end: 2620
  phase1_weight: 33.4
  phase2_weight: 20.43
  phase3_weight: 42.29
  phase4_weight: 3.88


global:
  max_concurrent: 10


jobs:
Reference

  - name: MiServer
    max_plots: 999
    farmer_public_key: 
    pool_public_key: 
    temporary_directory: d:\
    temporary2_directory:
    destination_directory: e:\
    size: 32
    bitfield: true
    threads: 8
    buckets: 128
    memory_buffer: 4000
    max_concurrent: 4
    max_concurrent_with_start_early: 4
    stagger_minutes: 35
    max_for_phase_1: 4
    concurrency_start_early_phase: 4
    concurrency_start_early_phase_delay: 10
    temporary2_destination_sync: false

**Comment by swar** [comment]
Your pool and farmer keys are not correct

**Comment by zhangyix19** [comment]
Hey, bro!
I've faced the same problem as yours and solved it.
Check if your pool and farmer keys have '0x' at their heads.
If so, delete '0x's and it can work !
